src/providers/EtherspotContextProvider.tsx

```diff
@@ -511,6 +512,41 @@ const EtherspotContextProvider = ({
     [sdk, accountAddress]
   );
 
+  // get ENS Node
+  const getENSNode = useCallback(
+    async (
+      chainId: number,
+      address: string | null = accountAddress,
+      recompute: boolean = true
+    ): Promise<ENSNode | { errorMessage?: string }> => {
+      const sdk = getSdkForChainId(chainId);
+
+      if (!sdk) return {};
```

Comment:
can we return nullish value instead of object? because in the code below you are even doing ternary check on which `{}` will pass as true so please change return to null or consider using `errorMessage` that you have in your return, speaking of which, if you are not planning to use `errorMessage` – remove it

Reply:
yes sure.